api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -104,7 +120,8 @@ def publish_cdr(cmd_name, args)
   common = Common.new
   env = ENVIRONMENTS[op.opts.project]
   account = env.fetch(:publisher_account)
-  app_sa = "#{ENVIRONMENTS[op.opts.project]}@appspot.gserviceaccount.com"
+  app_sa = "#{op.opts.project}@appspot.gserviceaccount.com"
```

Comment:
I think I broke this in #4447 - now it doesn't try to use the whole JSON structure as an email...